### PR TITLE
Mitigate AGOL ETL 404 errors

### PR DESCRIPTION
**Pull request by johnclary**
## Associated issues
- https://github.com/cityofaustin/atd-data-tech/issues/14612

## Testing

You can follow the [readme](https://github.com/cityofaustin/atd-moped/tree/main/moped-etl/arcgis) to run this ETL. it is best to use your local env after downloading a snapshot of production.

The ETL should work, while occasionally logging failures + retries.

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable

### CODE REVIEW

File: moped-database/views/component_arcgis_online_view.sql
Comment (johnclary) [inline moped-database/views/component_arcgis_online_view.sql:1]: i am squeezing in this change to increase the point radius from 5 meters to 7 meters.

**Comment by mddilley** [comment]
@johnclary just had this bomb out during the combined features replacement with 5 retries. Going to test it out with 10 to see how it goes!

**Comment by johnclary** [comment]
@mddilley thank you. and booooooo. i wonder if increasing the sleep to 5 seconds might actually be more productive...

**Comment by mddilley** [comment]
Ok, made it through with 10 retries, and it only made it to 4 attempts once through the whole script. Mostly 1 or 2 retries. Going to test out the longer sleep now.

**Review by mddilley — APPROVED** [review]
Worked great with the increased retries! Ship ship ship. 🚀

**Pull request merged**